cron/3.queueProcess.nolock.php

```diff
@@ -108,6 +108,11 @@
         $kill['npc'] = isNPC($kill);
         $kill['awox'] = ($kill['npc'] == true) ? false : isAwox($kill);
         $kill['solo'] = ($kill['npc'] == true) ? false : isSolo($kill);
+        
+        $capitalShipTypeIds = array(
```

Comment:
Should this be refactored elsewhere? I did not see any obvious way to map the items in `setup/items.json` into a list of IDs that correspond to capital ships, so I've added them directly here.